Classes/common/HTTP/CDTURLSessionInterceptorContext.m

```diff
@@ -0,0 +1,29 @@
+//
+//  CDTURLSEssionFilterContext.m
+//  
+//
+//  Created by Rhys Short on 17/08/2015.
+//
+//
+
+#import "CDTURLSEssionInterceptorContext.h"
+
+@implementation CDTURLSessionInterceptorContext
+
+-(instancetype) init {
```

Comment:
I tried building without providing an implementation for `init` and it worked - so maybe this can be deleted all together?